### PR TITLE
Update d_nes.cpp

### PR DESCRIPTION
**Pull request by Charles32k**
1 new game and 2 hacks

### CODE REVIEW

File: src/burn/drv/nes/d_nes.cpp [hunk 1]
```diff
@@ -17326,6 +17326,61 @@ struct BurnDriver BurnDrvnes_nekkestrbasc = {
...
+{ "a ressha de ikou (t -eng).nes",          147472, 0xfd7bc532, BRF_ESS | BRF_PRG },
+};
+
+STD_ROM_PICK(nes_atrain)
+STD_ROM_FN(nes_atrain)
+
+struct BurnDriver BurnDrvnes_atrain = {
+"nes_atrain", NULL, NULL, NULL, "1989 ?",
+"A Ressha de Ikou (T -Eng)\0", NULL, "Nintendo", "Miscellaneous",
+NULL, NULL, NULL, NULL,
```
Comment (dinkc64) [inline src/burn/drv/nes/d_nes.cpp:17360]: Hi,  please fix little type-o above? :)  (T-Eng)

Comment (Charles32k) [reply src/burn/drv/nes/d_nes.cpp:17360]: (T -Eng) ->(T-Eng)?

Comment (dinkc64) [reply src/burn/drv/nes/d_nes.cpp:17360]: thanks, yes :)

Comment (Charles32k) [reply src/burn/drv/nes/d_nes.cpp:17360]: thanks to you!

**Pull request merged**